app/index.html

```diff
@@ -20,6 +20,8 @@
     <script src="assets/vendor.js"></script>
     <script src="assets/ember-crud.js"></script>
```

Comment:
change this ember-crudddd.js


```suggestion
    <script src="assets/ember-crudddd.js"></script>
```